### PR TITLE
Add functionality to view assigned policies of a client, delete them and delete policies itself.

### PR DESCRIPTION
**Pull request by Ssatu**
New commands added:

1. deletePolicy (index of Policy)
- Command also deletes all corresponding assigned policies in relevant persons.

2. deleteAssigned (index of person) (index of assignedPolicy)

3. listAssigned (index of person)
- Lists assigned policies in CLI

### CODE REVIEW

File: src/main/java/longtimenosee/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
  */
 public class EditCommand extends Command {
 
-    public static final String COMMAND_WORD = "edit";
+    public static final String COMMAND_WORD = "editPerson";
```
Comment (loyhongshenggg) [inline src/main/java/longtimenosee/logic/commands/EditCommand.java:38]: Shall we change this to editContact?

File: src/main/java/longtimenosee/logic/commands/PersonCommand.java [hunk 2]
```diff
@@ -7,7 +7,7 @@
  */
 public class PersonCommand extends Command {
 
-    public static final String COMMAND_WORD = "showClient";
+    public static final String COMMAND_WORD = "clients";
```
Comment (loyhongshenggg) [inline src/main/java/longtimenosee/logic/commands/PersonCommand.java:10]: Shall we change _clients_ to _contacts_ too for standardization? What are your thoughts?

**Comment by loyhongshenggg** [comment]
Did some light testing, LGTM. Let me know your thoughts on the above! ☺️

**Comment by Ssatu** [comment]
> Did some light testing, LGTM. Let me know your thoughts on the above! ☺️

Thanks for the update! Made the changes you spotted :)

**Pull request merged**